container/api/src/test/java/org/wildfly/swarm/container/JARArchiveTest.java

```diff
@@ -1,108 +0,0 @@
-/**
```

Comment:
This and subsequent tests for previous container api classes have been removed but not added to :spi repository. Oversight?